srcs/go/libkungfu-comm/cmds.go

```diff
@@ -1,15 +1,65 @@
 package main
 
 import (
-	"os"
-
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
 	"github.com/lsds/KungFu/srcs/go/cmd/kungfu-run/app"
+	"net"
+	"net/http"
+	"os"
+	"strconv"
+	"strings"
 )
 
 import "C"
 
+var serverip string
```

Comment:
could you move the changes to a new file and leave `cmds.go` unchanged.

Reply:
Done